.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+* **Please check if the PR fulfills these requirements**
+- [ ] Indentation settings of this repository were used for the files of the commit(s)
```

Comment:
Ups, totally forgot. That's what we must do, yeah. Unfortunately, Eclipse does not provide an easy exportable file for XML editor settings like for Java. 

Let's keep this PR open for the moment.

Reply:
Perhaps it would be enough to just specify what kind of indentation we use. (e.g., no tabs, 2 spaces). As I guess Eclipse is not the default XML editor for most people, using a formatter profile is therefore perhaps unnecessary.